dev-docs/bidders/pixad.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: bidder
+title: Pixad
+description: Prebid Pixad Bidder Adapter.
+pbjs: true
+pbs: false
+biddercode: pixad
+media_types: banner,video
+gdpr_supported: false
+usp_supported: false
+coppa_supported: false
+schain_supported: false
+dchain_supported: false
+userIds: criteo, id5Id, sharedId, unifiedId
+safeframes_ok: true
+floors_supported: false
+multiformat_supported: will-bid-on-any
```

Comment:
You're missing a very important field here -- since pixad is an alias of admatic, you need this:
```
aliasCode: admatic
```

Reply:
Hi @bretg ,
Thank you for feedback.
I added code line. 👍 Best.